avif-coder/src/main/java/com/radzivon/bartoshyk/avif/coder/HeifCoder.kt

```diff
@@ -31,14 +35,15 @@ class HeifCoder {
         return decodeImpl(byteArray, scaledWidth, scaledHeight)
     }
 
-    fun encodeAvif(bitmap: Bitmap): ByteArray {
-        return encodeAvifImpl(bitmap)
+    fun encodeAvif(bitmap: Bitmap, quality: Int): ByteArray {
```

Comment:
If you wish to add parameter `quality` then it must be set default value equal to value in previous version to save compatibility

Reply:
Okay